_assets/stylesheets/pages/_tv.scss

```diff
@@ -0,0 +1,42 @@
+.tv-btns {
+  width: 182px;
+  height: 54px;
+}
+
+.tv-btns-container {
+  width: 500px;
+  margin: auto;
+}
```

Comment:
Remove specified with. Mobile screenshot below
![Screen Shot 2020-11-20 at 2 19 22 PM](https://user-images.githubusercontent.com/32345656/99840925-65728180-2b3b-11eb-9c61-99e1c4dfeae6.png)

Reply:
Look at the comment below

Reply:
Needs to be max-width instead to fix the issue you mentioned before.